### PR TITLE
Chore/improve paths

### PR DESCRIPTION
**Pull request by Sharqiewicz**
### What:

The paths in the app are just `strings` that can cause many problems, such as misspelling, pointing to a non-existent page (after renaming).

### How:

✅  Change the paths to be included in the `PATHS` enum. Enum is used for redirection throughout the application.
✅ Remove unused Stats page
✅ Fixed error appearing in the paths that only has the network name like: 
`portal.pendulumchain.org/amplitude` `portal.pendulumchain.org/pendulum` `portal.pendulumchain.org/foucoco` - now it's redirecting to the dashboard

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -14,18 +14,48 @@ import { TenantName } from './models/Tenant';
...
+export const PAGES_PATHS = {
+  DASHBOARD: 'dashboard',
+  GAS: 'gas',
+  BRIDGE: `${PATHS.SPACEWALK}/${PATHS.BRIDGE}`,
+  TRANSACTIONS: `${PATHS.SPACEWALK}/${PATHS.TRANSACTIONS}`,
+  NABLA: 'nabla',
+  NABLA_SWAP: `${PATHS.NABLA}/${PATHS.NABLA_SWAP}`,
+  NABLA_SWAP_POOLS: `${PATHS.NABLA}/${PATHS.NABLA_SWAP_POOLS}`,
+  NABLA_BACKSTOP_POOLS: `${PATHS.NABLA}/${PATHS.NABLA_BACKSTOP_POOLS}`,
+  STAKING: 'staking',
```
Comment (ebma) [inline src/app.tsx:51]:
```suggestion
  DASHBOARD: `${PATHS.DASHBOARD}`,
  GAS: `${PATHS.GAS}`,
  BRIDGE: `${PATHS.SPACEWALK}/${PATHS.BRIDGE}`,
  TRANSACTIONS: `${PATHS.SPACEWALK}/${PATHS.TRANSACTIONS}`,
  NABLA: `${PATHS.NABLA}`,
  NABLA_SWAP: `${PATHS.NABLA}/${PATHS.NABLA_SWAP}`,
  NABLA_SWAP_POOLS: `${PATHS.NABLA}/${PATHS.NABLA_SWAP_POOLS}`,
  NABLA_BACKSTOP_POOLS: `${PATHS.NABLA}/${PATHS.NABLA_BACKSTOP_POOLS}`,
  STAKING: `${PATHS.STAKING}`,
```

File: src/pages/bridge/Issue/index.tsx [hunk 2]
```diff
@@ -186,7 +187,7 @@ function Issue(props: IssueProps): JSX.Element {
         onClose={() => setConfirmationDialogVisible(false)}
         onConfirm={() => {
           setConfirmationDialogVisible(false);
-          navigateTo(`/${tenantName}/spacewalk/transactions`);
+          navigateTo(`/${tenantName}${PAGES_PATHS.TRANSACTIONS}`);
```
Comment (ebma) [inline src/pages/bridge/Issue/index.tsx:190]:
```suggestion
          navigateTo(`/${tenantName}/${PAGES_PATHS.TRANSACTIONS}`);
```

File: src/pages/bridge/Redeem/ConfirmationDialog.tsx [hunk 3]
```diff
@@ -56,7 +57,7 @@ export function ConfirmationDialog(props: ConfirmationDialogProps): JSX.Element
       <Button
         color="primary"
         onClick={() => {
-          navigateTo(`/${tenantName}/spacewalk/transactions`);
+          navigateTo(`/${tenantName}${PAGES_PATHS.TRANSACTIONS}`);
```
Comment (ebma) [inline src/pages/bridge/Redeem/ConfirmationDialog.tsx:60]:
```suggestion
          navigateTo(`/${tenantName}/${PAGES_PATHS.TRANSACTIONS}`);
```

**Review by ebma — CHANGES_REQUESTED** [review]
Nice changes 👍  The app seems broken though. If you open the deploy preview, you are not able to close the ToC dialog because the app is unresponsive, see the following error.

image

**Comment by Sharqiewicz** [comment]
@ebma I have implemented dynamic imports with dynamic paths using const pages = `import.meta.glob('./pages/**/index.tsx')`; from **ViteJS** ⚡ . This approach detects and registers all the files that match the specified pattern without fetching them immediately.

✨ Key Improvements:

- ✨ The directory structure for paths has been reorganized, making it more intuitive to create new page directories.
- ✨ The directories now match the paths configuration in app.tsx. For instance, the path `:network:/spacewalk/transactions` corresponds to `/pages/spacewalk/transactions/index.tsx`, which is automatically detected by `import.meta.glob` from ViteJS.
- ✨ All links in the app have been updated to use the `PATHS enum` from app.tsx, minimizing the risk of misspellings.

File: src/components/Layout/links.tsx [hunk 4]
```diff
@@ -74,19 +75,19 @@ export const links: Links = ({ tenantName }) => [
     suffix: <ExternalIcon />,
   },
   {
-    link: './spacewalk',
+    link: '/spacewalk',
```
Comment (ebma) [inline src/components/Layout/links.tsx:78]: Should we also replace this link with something?

Comment (ebma) [reply src/components/Layout/links.tsx:78]: As well as the nabla one in l. 96

**Review by ebma — APPROVED** [review]
Cool, I like it 👌

**Comment by Sharqiewicz** [comment]
@ebma changed ✅

**Review by ebma — APPROVED** [review]
LGTM

**Pull request merged**